pybamm/simulation.py

```diff
@@ -417,38 +462,69 @@ def solve(
             self._solution = None
             # Step through all experimental conditions
             inputs = inputs or {}
-            pybamm.logger.info("Start running experiment")
```

Comment:
why was this removed?

Reply:
It was removed by mistake. I will add this line in next commit.